test/parallel/test-worker-load-file-with-extension-other-than-js.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+
+const { Worker } = require('worker_threads');
+
+(common.mustCall(() => {
+  new Worker(fixtures.path('worker-script.ts'));
```

Comment:
Ummm, does this mean worker-script would run as CommonJS?

Reply:
Yes, it will be loaded as a CJS module given current implementation.